src/remote/addr/mod.rs

```diff
@@ -91,10 +95,17 @@ impl RemoteAddr {
         unimplemented!("So far, it is not possible to use this method!")
     }
 
-    pub fn send<T: RemoteMessage + Serialize>(&self, _msg: T) {
-        unimplemented!(
-            "So far, it is not possible to receive responses from remote destinations as futures!"
-        )
+    pub fn send<T, R, H>(&self, msg: T, c: &Addr<H>) -> Receiver<Box<dyn Any + Send>>
```

Comment:
Can you explain what `c` does? Is it just storing the `Responses`?

Reply:
Yes. Actor should store `Sender`, and use it whenever message of corresponding TypeId received